mappings/net/minecraft/world/World.mapping

```diff
@@ -77,6 +79,7 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 		COMMENT <p>Debug worlds are not modifiable and are typically meant for development and debug use only.
 		COMMENT See <a href="https://minecraft.gamepedia.com/Debug_mode">the minecraft wiki</a> as well.
 	METHOD method_27983 getRegistryKey ()Lnet/minecraft/class_5321;
+	METHOD method_31592 getEntityManager ()Lnet/minecraft/class_5577;
```

Comment:
How is this a manager? Seems more like a collection from the uses

Reply:
According to #1905, the return type is definitely [not an entity manager](https://github.com/FabricMC/yarn/blob/e9f0cea6e77e28bcdbb49e80ec95d0392107c54d/mappings/net/minecraft/world/entity/EntityIdMap.mapping)